tests/nnstreamer_filter_tensorflow/runTest.sh

```diff
@@ -21,11 +21,60 @@ if [[ -z "${TEST_TENSORFLOW}" ]]; then
     report
 fi
 
+if [[ ! $(pkg-config --libs tensorflow) ]]; then
+    report
+    exit
+fi
+
 # Test with mnist model
 PATH_TO_PLUGIN="../../build"
 PATH_TO_MODEL="../test_models/models/mnist.pb"
 PATH_TO_DATA="data/9.raw"
 
+if [[ -d $PATH_TO_PLUGIN ]]; then
```

Comment:
I think that, after this PR we can remove TEST_TENSORFLOW (line 20), defined to enable tensorflow testcases in nnstreamer.spec.

Reply:
Done!